src/site/apt/user_get_started_with_jsoniq.apt

```diff
@@ -0,0 +1,97 @@
+~~ Licensed to the Apache Software Foundation (ASF) under one or more
+~~ contributor license agreements.  See the NOTICE file distributed with
+~~ this work for additional information regarding copyright ownership.
+~~ The ASF licenses this file to You under the Apache License, Version 2.0
+~~ (the "License"); you may not use this file except in compliance with
+~~ the License.  You may obtain a copy of the License at
+~~
+~~     http://www.apache.org/licenses/LICENSE-2.0
+~~
+~~ Unless required by applicable law or agreed to in writing, software
+~~ distributed under the License is distributed on an "AS IS" BASIS,
+~~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~~ See the License for the specific language governing permissions and
+~~ limitations under the License.
+
+JSONiq extension to XQuery
+
+  The {{{http://jsoniq.org/docs/JSONiqExtensionToXQuery/html-single/index.html}JSONiq}} extension to XQuery allows processing XML and JSON natively and with a single language.
+
+
+* Example
+
+  This example uses the XML file (books.xml) from {{{http://www.w3schools.com/xsl/xquery_intro.asp}W3Schools XQuery Tutorial}}
+
+  [[i]] Query:
+
+----------------------------------------
+[
+	for $book in doc("books.xml")/bookstore/book
+	return
+	{
+		"category": data($book/@category),
+		"title" : data($book/title),
+		"author" : data($book/author)
+		"year" : data($book/year),
+		"price": data($book/price)
+	}
+]
+----------------------------------------
+
+  [[i]] Result:
+
+----------------------------------------
+[
+   {
+      "category":"COOKING",
+      "title":"Everyday Italian",
+      "author":"Giada De Laurentiis",
+      "year":"2005",
```

Comment:
Did you run this query?
Do you think we should return year as an integer and price as a double? It would seem to make more sense for your second example query where you are comparing "29.99" > 30. types could both be numeric.